plugins/destination/postgresql/client/migrate.go

```diff
@@ -113,11 +129,24 @@ func (c *Client) autoMigrateTable(ctx context.Context, table *schema.Table) erro
 			if _, err := c.conn.Exec(ctx, sql); err != nil {
 				return fmt.Errorf("failed to add column %s on table %s: %w", col.Name, table.Name, err)
 			}
-		default:
```

Comment:
I think this `default` case was a bug, and the PK check really should always be done, not only when there are no existence / type mismatches.